src/main/java/uk/gov/hmcts/reform/sscs/ccd/domain/SscsHearingType.java

```diff
@@ -0,0 +1,37 @@
+package uk.gov.hmcts.reform.sscs.ccd.domain;
+
+import com.fasterxml.jackson.annotation.JsonEnumDefaultValue;
+import com.fasterxml.jackson.annotation.JsonValue;
+import lombok.Getter;
+
+@Getter
+public enum SscsHearingType {
+
+    SUBSTANTIVE("substantive", "BBA3-substantive", "Substantive", "BBA3"),
+    DIRECTIONHEARINGS("directionHearings", "BBA3-directionHearings", "Direction Hearings", "BBA3"),
+    CHAMBERSOUTCOME("chambersOutcome", "BBA3-chambersOutcome", "Chambers Outcome", "BBA3"),
+    @JsonEnumDefaultValue
+    UNKNOWN("unknown", "unknown", "Unknown", "unknown");
+
+    private final String type;
+    private final String key;
+    private final String descriptionEn;
+    private final String serviceCode;
+
+    SscsHearingType(String type, String key, String descriptionEn, String serviceCode) {

```

Comment:
Can you use Lombok to keep things consistent?

Reply:
Resolved in a previous comment